packages/jellyfish-transaction/src/script/dftx/dftx_icxorderbook.ts

```diff
@@ -21,7 +21,7 @@ export interface ICXCreateOrder {
   orderType: number // -------------| 1 byte unsigned, 0x1 (INTERNAL) | 0x2 (EXTERNAL)
   tokenId: number // ---------------| VarUInt{1-9 bytes}
   ownerAddress: Script // ----------| n = VarUInt{1-9 bytes}, + n bytes
-  receivePubkey?: string // --------| 1 byte for len + len bytes
+  receivePubkey?: string // --------| n = VarUInt{1-9 bytes} (not optional, defaults to 0x00) + n bytes
```

Comment:
@Jouzo since it is public key, could it be always 0x00 (1 byte) or 0x21.... (33 bytes)?

Reply:
0x00 (default when undefined) | 0x21 (COMPRESSED_PUBLIC_KEY_SIZE) | 0x41 (PUBLIC_KEY_SIZE)

Reply:
right, I will indicate it, will make it clearer